src/FlowSimulation/FlowSimulation.jl

```diff
@@ -0,0 +1,9 @@
+export FlowSimulation
+
+module FlowSimulation
+
+    using FwiFlow
```

Comment:
Would be good to throw some kind of warning that this is quite overkill and that this will be replace by a proper non-tensorflow implementation ideally at some point.